.github/workflows/build-and-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8]
```

Comment:
Did we officially drop support for 3.7? 

I think skyportal and baselayer require >=3.7 at this point.

Reply:
So, the tuple unpacking syntax that I am using below was apparently introduced in Python 3.8. Is this OK?

I was emboldened to require Python 3.8 by this exchange on Slack in mid July:

@lpsinger : What Python versions do we target?

@dannygoldstein : 3.7+

@stefanv : Probably 3.8 now.

Reply:
For now we are requiring 3.7, so I request that we maintain compatibility with that until SP and B drop 3.7 support.